### PR TITLE
Merged till Dev

### PR DESCRIPTION
**Pull request by saniketprdxn**
Merged till Dev

### CODE REVIEW

File: assets/js/script.js [hunk 1]
```diff
@@ -1,26 +1,75 @@
...
+const coin = ['heads', 'tails'];
+const playerSelection = document.querySelector('.player-selection');
+const computerSelection = document.querySelector('.computer-selection');
+const playerScore = document.querySelector('.player-score');
+const computerScore = document.querySelector('.computer-score');
+const coinImg = document.querySelector('.coin img');
+const coinFig = document.querySelector('.coin figure');
+const btns = Array.from(document.querySelectorAll('.toss-call li'));
+
+function init(btns,playerScore,computerScore,playerSelection,computerSelection) {
```
Comment (anikeshb) [inline assets/js/script.js:12]: no comment for the function???

File: assets/js/script.js [hunk 2]
```diff
@@ -1,26 +1,75 @@
...
+btns.forEach((btn) => btn.addEventListener('click',flip));
+playerScore.innerText = "0";
+computerScore.innerText = "0";
+playerSelection.innerText = "nothing";
+computerSelection.innerText = "nothing";
+}
+
+init(btns,playerScore,computerScore,playerSelection,computerSelection);
+
+function flip(){
```
Comment (anikeshb) [inline assets/js/script.js:22]: no comment for this function??

File: assets/js/script.js [hunk 3]
```diff
@@ -1,26 +1,75 @@
...
+	playerSelection.innerText = this.innerText;
+	if(playerSelection.innerText === coin[0]) {
+			computerSelection.innerText = coin[1];
+	} else {
+			computerSelection.innerText = coin[0];
+	}
+	let coinFlip = coin[Math.floor(Math.random()*coin.length)];
```
Comment (anikeshb) [inline assets/js/script.js:29]: explain with comment why did u do this??

File: assets/js/script.js [hunk 4]
```diff
@@ -1,26 +1,75 @@
...
+	coinImgFlip(coinFlip);
+}
+
+function coinImgFlip(flipAnswer) {
```
Comment (anikeshb) [inline assets/js/script.js:33]: comment ??

File: assets/js/script.js [hunk 5]
```diff
@@ -1,26 +1,75 @@
...
+	if(flipAnswer === coin[0]) {
+		coinImg.src = "./assets/Images/"+coin[0]+".jpg";
+	} else {
+		coinImg.src = "./assets/Images/"+coin[1]+".jpg";
+	}
+	setTimeout(() => {
+		coinFig.classList.remove('flipping');
+		updatescore(flipAnswer);
+	},1300)
+}
```
Comment (anikeshb) [inline assets/js/script.js:44]: while this is happening, we may use gif instead.

File: assets/js/script.js [hunk 6]
```diff
@@ -1,26 +1,75 @@
...
+
+function updatescore(coinFlipAns) {
```
Comment (anikeshb) [inline assets/js/script.js:46]: camel case naming convention to be followed.

File: assets/js/script.js [hunk 7]
```diff
@@ -1,26 +1,75 @@
...
+		computerScore.innerText++;
+	}
+	if(playerScore.innerText == 5 ) {
+		win(playerScore);
+	}	else if(computerScore.innerText == 5) {
+		win(computerScore);
+	}
+}
+
+function win(winner) {
```
Comment (anikeshb) [inline assets/js/script.js:59]: comments for what the function does???

**Comment by anikeshb** [comment]
Please use comments to explain your code better.

File: assets/js/script.js (hunk 5 above)
Comment (saniketprdxn) [reply assets/js/script.js:49]: i will use it in the todays task

**Pull request merged**